server/src/app/modules/auth/auth.service.ts

```
@@ -127,11 +127,21 @@
 }; 
  
 
+const googleAuthLogin = async(payload:IUser)=>{
+    let user = await UserModel.findOne({email: payload.email});
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'user' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)